gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,655 @@
+            "precision": 2,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "conditional_formats": [],
+                        "q": "avg:gitlab.transaction.event_create_repository_total.1{*}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Repositories Created",
+            "type": "query_value",
+            "viz": "query_value",
+            "width": 16,
+            "x": 96,
+            "y": 25
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 9,
+            "isShared": false,
+            "precision": 1,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "precision": 1,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "conditional_formats": [],
+                        "q": "avg:gitlab.transaction.event_fork_repository_total.1{*}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Repositories Forked",
+            "type": "query_value",
+            "viz": "query_value",
+            "width": 16,
+            "x": 96,
+            "y": 13
+        },
+        {
+            "add_timeframe": true,
+            "auto_refresh": false,
+            "background_color": "gray",
+            "bgcolor": "gray",
+            "content": "Repositories",
+            "font_size": "18",
+            "height": 10,
+            "html": "Repositories",
+            "isShared": false,
+            "refresh_every": 30000,
+            "scaleFactor": 1,
+            "show_tick": true,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 16,
+            "x": 96,
+            "y": 1
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 12,
+            "isShared": false,
+            "legend": false,
+            "markers": [],
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.transaction.event_push_commit_total{*}",
```

Comment:
This metric is submitted as a monotonic counter, can we use a bar graph here instead?